shared/Microsoft.AspNetCore.RangeHelper.Sources/RangeHelper.cs

```diff
@@ -16,21 +18,32 @@ namespace Microsoft.AspNetCore.Internal
     /// </summary>
     internal static class RangeHelper
     {
+        // Is temporary to avoid build break
```

Comment:
What's the plan to remove this? After reacting to this change? (Just want to make sure we don't release with this in.)

Reply:
Sorry @muratg , didn't notice your comment. I am going to revert this temporary change after I merge MVC PR https://github.com/aspnet/Mvc/pull/7163. Also this class in internal

Reply:
Sounds good, thanks Kiran!